examples/simple-example/README.md

```diff
@@ -0,0 +1,37 @@
+<div id="top"></div>
+
+## Simple Example
+
+*This simple example project demonstrates the use of miranum-core. Here we have created a fairly simple process that is automated using 
+miranum-core. The project consists of three modules:*
+- [simple-example_camunda-7](simple-example-camunda-7)
+- [simple-example_camunda-8](simple-example-camunda-8)
+- [simple-example-core](simple-example-core)
+
+*The application and domain logic is implemented in the `simple-example-core` project. It contains the services that are called by a workflow engine.*
```

Comment:
Vll machts Sinn, die Architektur mit mermaid zu visualisieren? Was hälst du davon?

Hab in dem PR die Zielarchitektur vom Projekt aufgezeichnet. Finds aber nicht ganz die richtige Stelle in der main-Readme. Hast du vll ne bessere Idee?
https://github.com/FlowSquad/miranum/pull/12/files

Reply:
Ich finde es tatsächlich in der Main nicht ganz schlecht als Überblick über die Architektur. Die dient ja allgemein als Einstiegspunkt für jemanden der sich für das Produkt interessiert. Ist zumindest das, was ich in nem Repo als erstes lese. Es mag vielleicht aktuell weil die main-README noch sehr mager aussieht, komisch aussehen, aber ich denke das wird schon so passen.

Reply:
Alles klar. Ich frag dich in dem PR mal als Reviewer an :) Dann können wir den auch mergen